### ISSUE TITLE
IndexError: list index out of range

### ISSUE THREAD

**Issue by aheath70** [bremor/bureau_of_meteorology#28]
Hassio - 0.117.6 running on Rasberry Pi 3B+
Bureau of Meteorology custom component 0.8.1 installed via HACS

Getting the following error:

```
2020-11-14 16:37:54 ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 616, in _update_entity_states
    await asyncio.gather(*tasks)
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 283, in async_update_ha_state
    self._async_write_ha_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 320, in _async_write_ha_state
    sstate = self.state
  File "/config/custom_components/bureau_of_meteorology/sensor.py", line 155, in state
    state = self.collector.daily_forecasts_data["data"][self.day][self.sensor_name]
IndexError: list index out of range
```

**Comment by bremor** [comment]
Thanks Andrew :-)

**Comment by bremor** [comment]
Fixed in [0.8.3](https://github.com/bremor/bureau_of_meteorology/releases/tag/0.8.3)
